src/openms/source/FORMAT/DATAACCESS/CsiFingerIdMzTabWriter.cpp

```diff
@@ -63,7 +63,18 @@ void CsiFingerIdMzTabWriter::read(const std::vector<String> & paths, Size number
 
     ifstream file(pathtocsicsv);
 
-    if (file)
+    int number_of_lines = 0;
+
+    if (file) 
+    {
+        std::string line;
+        while (std::getline(file, line))
```

Comment:
will this walk to the end of th efile ?

Reply:
yes should - is not needed hm? Only need to check if there are more than one file.